modules/github/Makefile.init

```diff
@@ -17,16 +17,21 @@ GITHUB_TERRAFORM_TEMPLATES = .github/workflows/chatops.yml \
 	.github/mergify.yml \
 	.github/renovate.json
 
+# Create a file (contents unimportant) with this name
+# to prevent pr/auto-format etc. from updating the GitHub templates.
+GITHUB_UPDATE_DISABLE_SENTINEL := .github/.github-update-disabled
```

Comment:
What I don't like about this is that it's not specific to any workflow. It's like a ENV that we have no idea what consumes it. What about naming it after the workflow it disables?

```suggestion
GITHUB_UPDATE_DISABLE_SENTINEL := .github/.github-update-disabled
```